draft-demarco-oauth-status-assertions.md

```diff
@@ -676,6 +676,125 @@ may not always coincide with the actual usability of a Digital Credential,
 allowing Verifiers to examine and make educated conclusions based on a
 variety of scenarios.
 
+# Complex Status Assertions
+
+Status assertions can be complex, and are not limited to simple boolean information.
+This enables verifier policies to be conditioned on the presence of secured information, instead of the absence of information.
+This section proposes syntax to support complex assertions.
+The `validity` claim MUST be present and be either `true` or `false`.
+The `validity_reasons` claim MAY be present and if present MUST be an object.
+The semantics of the claims within the `validity_reasons` object are determined by the issuer.
+
+An example of a boolean status is:
+
+~~~
+{
+    "alg": "ES256",
+    "typ": "status-assertion+jwt",
+    "kid": "w8ZOZRcx21Zpry7H-0VLBsH7Wf7WXb6TeK3qVMCpY44"
+}
+.
+{
+    "iss": "https://issuer.example.org",
+    "iat": 1504699136,
+    "exp": 1504785536,
+    "credential_hash": "xnlAq6Ma8fgu1z4hdGphJnKLulaVHpLCFeZFUGpQ2dA",
+    "credential_hash_alg": "sha-256",
+    "validity": false,
+    "validity_reasons": {
```

Comment:
wdyt about

`validity` -> `credential_status_validity`
`validity_reason` -> `credential_status`

Reply:
where `credential_status_validity` is required, therefore all the other section must be updated accordingly